CHANGELOG.md

```diff
@@ -9,6 +9,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Fix LocalVideo not rendering on initial attempt
+
+### Added
+- Add JoinedFromAnotherDevice meeting status
+
+### Changed
+- Allow property passthough to IconButton in NavbarItem
```

Comment:
not seeing any code for this

Reply:
That was merged in another comment but the changelog hadn't been updated to add the 2.0 section, so I'm just correcting that here